Packs/TaniumThreatResponse/Playbooks/playbook-Tanium_Threat_ResponseV2_-_Create_Connection.yml

```diff
@@ -0,0 +1,200 @@
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Tanium Threat Response v2
+      description: Creates a local or remote connection.
+      id: f0a548a8-59c2-4b05-8afe-ed81e92ce5db
+      iscommand: true
+      name: Create Connection
+      script: Tanium Threat Response v2|||tanium-tr-create-connection
+      type: regular
+      version: -1
+    taskid: f0a548a8-59c2-4b05-8afe-ed81e92ce5db
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 195
+        }
+      }
+  "2":
+    id: "2"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    loop:
+      exitCondition: ""
+      iscommand: false
+      max: 0
+      wait: 1
+    nexttasks:
+      '#none#':
+      - "3"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      Ids:
+        simple: ${Tanium.Connection.id}
+      Interval:
+        simple: "1"
+      PollingCommandArgName:
+        simple: id
+      PollingCommandName:
+        simple: tanium-tr-list-connections
+      Timeout:
+        simple: "10"
+      dt:
+        simple: Tanium.Connection(val.status != 'connected')
+    separatecontext: true
+    skipunavailable: false
+    task:
+      brand: ""
+      description: |-
+        Use this playbook as a sub-playbook to block execution of the master playbook until a remote action is complete.
+        This playbook implements polling by continuously running the command in Step \#2 until the operation completes.
+        The remote action should have the following structure:
+
+        1. Initiate the operation.
+        2. Poll to check if the operation completed.
+        3. (optional) Get the results of the operation.
+      id: 0466579e-891a-4f2a-820a-90f03d8f805b
+      iscommand: false
+      name: GenericPolling
+      playbookId: GenericPolling
+      type: playbook
+      version: -1
+    taskid: 0466579e-891a-4f2a-820a-90f03d8f805b
+    timertriggers: []
+    type: playbook
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 370
+        }
+      }
+  "3":
+    id: "3"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 7822a39a-4735-4f58-893e-b8fe70f25c4c
+      iscommand: false
+      name: Done
+      type: title
+      version: -1
+      description: ''
+    taskid: 7822a39a-4735-4f58-893e-b8fe70f25c4c
+    timertriggers: []
+    type: title
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 545
+        }
+      }
+version: -1
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 550,
+        "width": 380,
+        "x": 50,
+        "y": 60
+      }
+    }
+  }
+tests:
+- Tanium Threat Response V2 Test
+- Tanium Threat Response Test
```

Comment:
why do we have both tests here?

Reply:
not sure, changed to the playbook that should test this playbook